docs/v3.x/deployment/ibm-cloud.md

```diff
@@ -0,0 +1,332 @@
+
+### IBM Cloud Install Requirements
+
+- You must have a [free IBM Cloud account](https://cloud.ibm.com/registration) before doing these steps.
+
+If you already have the IBM Cloud CLI installed locally on your computer. Skip to [Login to IBM Cloud](#_2-login-to-ibm-cloud-from-your-cli).
+
+#### 1. IBM Cloud CLI Installation
+
+Download and install the [CLI](https://cloud.ibm.com/docs/cli) for your operating system:
+
+:::: tabs
+
+::: tab "Mac O/S"
+Run the following from your terminal:
+
+```bash
+curl -sL https://ibm.biz/idt-installer | bash
+```
+
+:::
+
+::: tab Ubuntu
+Run the following from your terminal:
+
+```bash
+curl -sL https://ibm.biz/idt-installer | bash
+```
+
+:::
+
+::: tab Windows
+Run the following in Powershell as administrator:
+
+```bash
+[Net.ServicePointManager]::SecurityProtocol = "Tls12, Tls11, Tls, Ssl3"; iex(New-Object Net.WebClient).DownloadString('https://ibm.biz/idt-win-installer')
+```
+
+:::
+
+::::
+
+#### 2. Login to IBM Cloud from your CLI
+
+Next, you need to login to IBM Cloud from your computer.
+
+```bash
+ibmcloud login
+```
+
+Follow the instructions and return to your command line.
+
+#### 3. Create a new project (or use an existing one)
+
+Create a Strapi project with MongoDB [section of the documentation](../guides/databases.md#install-strapi-locally-with-mongodbb). But don't worry about the username and password for now. For convenience we´re going to use the URI instead.
+
+Create a [new Strapi project](../getting-started/quick-start.md) (if you want to deploy an existing project go to step 4).
+
+::: warning NOTE
+
+If you plan to use **MongoDB** with your project, [refer to the create a Strapi project with MongoDB section of the documentation](../guides/databases.md#install-mongodb-locally) then, jump to step 4.
+
+:::
+
+`Path: ./`
+
+:::: tabs
+
+::: tab yarn
+
+Use **yarn** to install the Strapi project (**recommended**). [Install yarn with these docs](https://yarnpkg.com/lang/en/docs/install/)
+
+```bash
+yarn create strapi-app my-project --quickstart
+```
+
+:::
+
+::: tab npx
+
+Use **npm/npx** to install the Strapi project
+
+```bash
+npx create-strapi-app my-project --quickstart
+```
+
+:::
+
+::::
+
+::: tip
+When you use `--quickstart` to create a Strapi project locally, a **SQLite database** is used which is not compatible with IBM Cloud. Therefore, another database option [must be chosen](#_7-ibm-cloud-database-set-up).
+:::
+
+#### 4. Update `.gitignore`
+
+Add the following line at end of `.gitignore`:
+
+`Path: ./my-project/.gitignore`
+
+```txt
+package-lock.json
+```
+
+Even if it is usually recommended to version this file, it may create issues on Cloud Foundry.
+
+#### 5. Create `.cfignore`
+
+Copy all the content of the `.gitignore` except the next line:
+
+`Path: ./my-project/.gitignore`
+
+```txt
+build
+```
+
+And add the next ones:
+
+```txt
+.editorconfig
+.env.example
+.gitignore
+manifest.yml
+README.md
```

Comment:
Is there a specific reason we are ignoring the README.md here? (I'm not super familiar with IBM's CF)

Reply:
Sure, here we put the files we don't want to push to the IBM Cloud Node.js instance.
- https://docs.cloudfoundry.org/devguide/deploy-apps/prepare-to-deploy.html#exclude

It's the same idea under Heroku's .slugignore file.
- https://docs.cloudfoundry.org/devguide/deploy-apps/prepare-to-deploy.html#exclude

Notes:
- On IBM Cloud is mandatory, Heroku uses .gitignore as fallback. In fact IBM CLI is not based on git, it pushes what you have in
- In both cases you need to put the whole list, they don't merge both files.

Reply:
Gotcha that makes sense